package-setup/setup-client.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+# exit if any command below fails
+set -e
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+YELLOW='\033[0;33m'
+WHITE='\033[0;37m'
+
+# Check if ROS is sourced
+
+if [ "$ROS_DISTRO" == "" ];
+then
+	echo "Installation cannot continue. No ROS sourced, please check if ROS is installed and sourced. Please try again after that!"
+	exit 0
+fi
+
+echo "Welcome to the support assistant for setting up the client PC Neobotix packages"
+
+uni_ans=""
+robot_model=""
+
+# Go to home directory
+cd ~
+
+mkdir -p client_workspace/src
+cd client_workspace/src
+
+while [[ "$robot_model" != "mp_400" && 
+        "$robot_model" != "mp_500" && 
+        "$robot_model" != "mpo_500" && 
+        "$robot_model" != "mpo_700" &&
+        "$robot_model" != "rox" ]]; do
+	echo -e "${GREEN} Choose your robot (mp_400/mp_500/mpo_500/mpo_700/rox)"
+
+	read robot_model
+
+	echo -e "${WHITE}"
+
+	if [ "$robot_model" == "rox" ]; then
+		git clone --branch $ROS_DISTRO     https://github.com/neobotix/rox.git || { true; echo -e ${YELLOW} "Continuing with the installation"; }
```

Comment:
@neopar what do you think about this line? 

In the case if the user decides to cancel the installation process halfway through and restart the process, he has to delete the folders and git packages that was already cloned, instead I use a `||` and check if we are good to continue. IS this way of `try and exception` is good ? or do you have any other idea to better handle this?